crates/bevy_pbr/src/prepass/mod.rs

```diff
@@ -508,8 +508,11 @@ where
             .mesh_key
             .intersects(MeshPipelineKey::NORMAL_PREPASS | MeshPipelineKey::DEFERRED_PREPASS)
         {
-            vertex_attributes.push(Mesh::ATTRIBUTE_NORMAL.at_shader_location(3));
             shader_defs.push("NORMAL_PREPASS_OR_DEFERRED_PREPASS".into());
+            if layout.0.contains(Mesh::ATTRIBUTE_NORMAL) {
```

Comment:
I feel like I'd want to see a warning if the NORMAL_PREPASS is enabled without normals

Reply:
I added a warning if the NORMAL_PREPASS is enabled without normals